kore/src/Kore/Internal/TermLike.hs

```diff
@@ -57,7 +56,6 @@ module Kore.Internal.TermLike
     , mkBottom
     , mkInternalBytes
     , mkInternalBytes'
-    , mkBuiltin
     , mkInternalBool
     , mkInternalInt
     , mkInternalString
```

Comment:
I can't comment on the specific lines, but shouldn't the `mkBuiltin...` functions for collections be renamed to `mkInternal...` as well?